pages/common/vimdiff.md

```diff
@@ -0,0 +1,16 @@
+# vimdiff
+
+> Open up to four files in vim and show their differences between them.
+> See `vim` documentation for working with files and navigating within a window.
+
+- Open two files and show the differences (up to four files can be compared):
+
+`vimdiff {{file1}} {{file2}}`
+
+- Open two files using a horizontal window split instead of the default vertical split:
+
+`vimdiff -o {{file1}} {{file2}}`
+
+- Move cursor to window left|right|up|down:
```

Comment:
I'd write this as "Move the cursor to the window on the left, right, top or bottom:" (but I'm open to other phrasings, or arguments to keep the current one).